### PR TITLE
Updated the app color scheme

### PR DESCRIPTION
**Pull request by ritik-bluebash**
Issue: #361 
Added the new colour on all pages.
Updated the icon, logo and png colours as well.
**1. Not signed in**
Screenshot from 2022-02-18 12-50-07

**2. Signed in page**
Screenshot from 2022-02-18 12-48-28

### CODE REVIEW

**Comment by duarme** [comment]
😅 Seeing it on the page, it's way more "electric" than we imagined. Please try a version using this primary color instead: **#006a9c** (you can wait on the images until we're sure!)

**Comment by ritik-bluebash** [comment]
Hello @duarme @ValerioVillani @vishal-bluebash 
I have tried these variants of the colour **#006a9c** from https://www.w3schools.com/colors/colors_picker.asp

**1. #006999**
Screenshot from 2022-02-22 11-22-27

**2. 009de6**
Screenshot from 2022-02-22 11-34-58

Please let me know which variant of this color will be perfect for the app.

**Comment by duarme** [comment]
@ritik-bluebash that definitely looks good 😄 . Question: are web-safe colors still relevant? I can be wrong but it seems they aren't important anymore, see: https://graphicdesign.stackexchange.com/questions/40/are-web-safe-colors-still-important can we also see a version with the original **#006a9c**, please?

**Comment by duarme** [comment]
I did one last comparison with #0077aa which is slightly lighter, on the left. 
Darker seems a bit more serious/reliable, while lighter seems a bit more easy/serene.

@ValerioVillani @ritik-bluebash @vishal-bluebash which one do you prefer?

CleanShot 2022-02-22 at 09 47 15

**Comment by ValerioVillani** [comment]
@duarme I like the darker one. 006699.

**Comment by vishal-bluebash** [comment]
I like this one too 006699 @duarme @ValerioVillani

**Comment by duarme** [comment]
Ok, then let's go with the one that includes a 69, and let's hope it's good luck! 😎

File: app/javascript/icons/fav-icon.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="16.1" height="16.1" viewBox="0 0 16.1 16.1"><defs><style>.cls-1{fill:none;}.cls-2{fill:#006699;}.cls-3{clip-path:url(#clip-path);}.cls-4{fill:#fff;}</style><clipPath id="clip-path" transform="translate(0.1 0.14)"><rect class="cls-1" x="-0.18" y="2.38" width="15.14" height="11.21"/></clipPath></defs><rect class="cls-2" width="16.1" height="16.1"/><g class="cls-3"><path class="cls-4" d="M10.54,8.88l-1.18-2-1.31.73L10.56,12l4.33-8.11H13.38Z" transform="translate(0.1 0.14)"/><path class="cls-4" d="M8.75,5.71A3.77,3.77,0,0,0,6.88,4.2,4,4,0,0,0,1.37,7.65,4.07,4.07,0,0,0,2,10.06a4,4,0,0,0,4.35,1.82,3.84,3.84,0,0,0,2.07-1.23l-.8-1.39A2.55,2.55,0,0,1,6,10.34,2.46,2.46,0,0,1,3,7.77,2.45,2.45,0,0,1,4.8,5.57a2.44,2.44,0,0,1,1.51.07,3.43,3.43,0,0,1,1.13.82Z" transform="translate(0.1 0.14)"/></g></svg>
```
Comment (duarme) [inline app/javascript/icons/fav-icon.svg:1]: Add newline at end of file, please. You can set up your editor/ide so that it does this automatically.

**Pull request merged**